examples/android/CHIPTest/app/src/main/cpp/CHIPTest-JNI.cpp

```diff
@@ -182,15 +169,20 @@ static void onLog(const char * fmt, ...)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
     jmethodID method;
-    jstring strObj = NULL;
-    char str[512]  = { 0 };
+    jstring strObj        = NULL;
+    jclass sTestEngineCls = NULL;
+    char str[512]         = { 0 };
     va_list args;
 
     ChipLogProgress(Test, "Received onLog");
     JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
     VerifyOrExit(env != NULL, err = CHIP_JNI_ERROR_NO_ENV);
 
```

Comment:
please add JniLocalReferenceScope scope(env); to release local ref in callback flow.